samples/python/hello_classification/hello_classification.py

```diff
@@ -8,7 +8,9 @@
 
 import cv2
 import numpy as np
-from openvino.inference_engine import IECore
+from openvino.runtime import Core, Tensor, Layout
+from openvino.runtime.impl import Type
```

Comment:
`impl` has been dropped in https://github.com/openvinotoolkit/openvino/pull/9096

Just import `Type` from runtime module.